examples/lighting-app/efr32/src/AppTask.cpp

```diff
@@ -214,13 +212,7 @@ void AppTask::AppTaskMain(void * pvParameter)
         // Otherwise, blink the LED ON for a very short time.
         if (sAppTask.mFunction != kFunction_FactoryReset)
         {
-            // Consider the system to be "fully connected" if it has service
-            // connectivity
-            if (sHaveServiceConnectivity)
```

Comment:
Seems to me that examples may still want to handle this case under the condition: `Thread provisioned + number of fabrics > 0` or so, but I agree this doesn't have to be part of this commit.